common/brave_farbling_constants.h

```diff
@@ -0,0 +1,15 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMMON_BRAVE_FARBLING_CONSTANTS_H_
+#define BRAVE_COMMON_BRAVE_FARBLING_CONSTANTS_H_
+
+enum BraveFarblingLevel {
```

Comment:
I think this makes more sense in components/brave_shields

Reply:
so actually per https://github.com/brave/brave-core/pull/5220/files#r408387679 maybe this makes more sense in `//brave/third_party/blink/renderer`